src/main/kotlin/com/terraformation/backend/tracking/db/T0PlotStore.kt

```diff
@@ -121,13 +151,32 @@ class T0PlotStore(
         // ensure no leftover densities for species that are not in this request
         dslContext.deleteFrom(this).where(MONITORING_PLOT_ID.eq(monitoringPlotId)).execute()
 
-        var insertQuery = dslContext.insertInto(this, MONITORING_PLOT_ID, SPECIES_ID, PLOT_DENSITY)
+        var insertQuery =
+            dslContext.insertInto(
+                this,
```

Comment:
Note that we're deleting rows in this plot before inserting, which kindof defeats the purpose of the modified fields. I could alternatively only delete ones that aren't in the list of species passed in, or could remove the modified column. I'm open to either.

Reply:
Sure, that works too.